src/Generators/Statements/ResourceGenerator.php

```diff
@@ -79,7 +92,7 @@ protected function populateStub(string $stub, Controller $controller, ResourceSt
             . ($controller->namespace() ? '\\' . $controller->namespace() : '');
 
         $stub = str_replace('{{ namespace }}', $namespace, $stub);
-        $stub = str_replace('{{ import }}', $resource->collection() ? 'Illuminate\\Http\\Resources\\Json\\ResourceCollection' : 'Illuminate\\Http\\Resources\\Json\\JsonResource', $stub);
```

Comment:
These were stringified as they are not available at runtime as part of the Laravel framework.